packages/graph/hash_graph/lib/graph/src/store/postgres/pool.rs

```diff
@@ -1,12 +1,28 @@
 use async_trait::async_trait;
+use bb8_postgres::{
+    bb8::{ErrorSink, Pool, PooledConnection, RunError},
+    PostgresConnectionManager,
+};
 use error_stack::{IntoReport, Result, ResultExt};
-use sqlx::{postgres::PgConnectOptions, ConnectOptions, PgPool};
-use tracing::log::LevelFilter;
+use tokio_postgres::{Client, Config, Error, GenericClient, NoTls, Transaction};
 
 use crate::store::{DatabaseConnectionInfo, PostgresStore, StoreError, StorePool};
 
 pub struct PostgresStorePool {
-    pool: PgPool,
+    pool: Pool<PostgresConnectionManager<NoTls>>,
+}
+
+#[derive(Debug, Copy, Clone)]
+struct ErrorLogger;
+
+impl ErrorSink<Error> for ErrorLogger {
+    fn sink(&self, error: Error) {
+        tracing::error!(%error, "Store pool has encountered an error");
```

Comment:
Perhaps being more explicit about the pool:
```suggestion
        tracing::error!(%error, "Store connection pool has encountered an error");
```